hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: minor
+
+This release adds the :func:`~hypothesis.extra.django.from_form` function, which allows automatic testing against Django forms. (:issue:`35`)
```

Comment:
:tada: 

Can you please also add a further line: "Thanks to Paul Stiverson for this feature, which resolves our oldest open issue!"

(It might feel a bit weird to write or copy-paste your own credits, but we think it's really important that contributors do get acknowledged... and we've automated ourselves out of the release process after this so it's up to you :rofl:)